src/ils.cpp

```diff
@@ -101,6 +101,66 @@ usage() {
         "Usage: ils [-ArlLv] dataObj|collection ... ",
         "Usage: ils --bundle [-r] dataObj|collection ... ",
         "Display data Objects and collections stored in irods.",
+        " ",
+        "The following is typical output for ils with no arguments:"
+        "    $ ils",
+        "    /tempZone/public:",
+        "      big",
+        "      foo",
+        "The only information displayed are the names of data objects in the current",
+        "working directory.",
+        " ",
+        "The long format (given by -l) will display some more information:",
+        "    $ ils -l",
+        "    /tempZone/public:",
+        "      rods              0 demoResc     41943040 2021-04-29.17:57 & big",
+        "      rods              2 comp;arch    40236581 2021-04-19.03:00 X big",
+        "      alice             2 repl1;child1          283 2021-04-29.17:54 ? foo",
+        "      alice             3 repl1;child2          283 2021-04-29.17:56 ? foo",
+        "      alice             4 repl1;child3          283 2021-04-29.17:57 ? foo",
+        "This displays replica information and system metadata for the replicas of",
+        "the data objects in the current working directory. This information includes",
+        "the owner of the data object and the replica's number, size, last modified",
+        "time, and status.",
+        " ",
+        "The very long format (given by -L) will display even more information:",
+        " ",
+        "    $ ils -L",
+        "    /tempZone/public:",
+        "      rods              0 demoResc     41943040 2021-04-29.17:57 & big",
+        "        sha2:gKNyEYjkAhiwiyZ3a8U72ugeR4T/9x1xRQoZcxnLoRM=    generic    /var/lib/irods/Vault/home/rods/big",
+        "      rods              2 comp;arch    40236581 2021-04-19.03:00 X big",
```

Comment:
same, bump the filesize lines to the right to align with the foos

Reply:
Done